src/app.js

```diff
@@ -1,22 +1,24 @@
 const express = require('express');
-
+require('dotenv').config();
+const bodyParser = require('body-parser');
+const connectToMongo = require('./db/connection');
 const router = require('./routes');
 const tutorRouter = require('./routes/tutor');
 const questionRouter = require('./routes/question');
-app.use('/', router);
-app.use('/tutors', tutorRouter);
-app.use('/questions', questionRouter);
 
-require('dotenv').config();
-const connectToMongo = require('./db/connection');
 const app = express();
 const port = process.env.NODE_LOCAL_PORT;
 
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
```

Comment:
I also used cookie-parser to handle cookies in the request but I think we should discuss with the rest of the team how we will implement the session because Emir used different pattern in his code.

Reply:
> Isn't it stateless auth Ehab? Because we are using jwt

Yes, but I assumed the token will be kept in the cookies. We can get the user every time from the request. you may see the onlyAuthenticated function that I wrote and then we can discuss how to implement the session.